sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample_AnalyzeOperation.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.AI.TextAnalytics.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.TextAnalytics.Samples
+{
+    [LiveOnly]
+    public partial class TextAnalyticsSamples: SamplesBase<TextAnalyticsTestEnvironment>
+    {
+        [Test]
+        public async Task AnalyzeOperation()
+        {
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+
+            var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+
+            #region Snippet:TextAnalyticsAnalyzeOperation
+            var batchDocuments = new List<TextDocumentInput>
+            {
+                new TextDocumentInput("1", "Microsoft was founded by Bill Gates and Paul Allen.")
+                {
+                     Language = "en",
+                }
+            };
+
+            AnalyzeOperationOptions operationOptions = new AnalyzeOperationOptions()
+            {
+                KeyPhrasesTaskParameters = new KeyPhrasesTaskParameters()
+                {
+                    ModelVersion = "latest"
```

Comment:
just curious, what happens if you don't set the `ModelVersion`?

Reply:
We get 

```
{
    "error": {
        "code": "InvalidArgument",
        "message": "Invalid entity recognition task in request"
    }
}
```

Reply:
plop .. and this is a service bug?
I wonder if instead of asking the user to do this is better if in the SDK we check.... so if the user didn't set the value, we "hardcode it" to `latest`. That way when the service fixes it, the user doesn't need to do anything new and they will never now

Reply:
this shouldn't be needed anymore